forc-plugins/forc-node/src/testnet/cmd.rs

```diff
@@ -0,0 +1,30 @@
+use crate::{
+    consts::{DEFAULT_PEERING_PORT, DEFAULT_PORT, TESTNET_BOOTSTRAP_NODE},
+    util::DbConfig,
+};
+use clap::Parser;
+use std::{net::IpAddr, path::PathBuf};
+
+#[derive(Parser, Debug, Clone)]
+pub struct TestnetCmd {
+    #[clap(long = "peer-id")]
+    pub peer_id: Option<String>,
+    #[clap(long = "secret")]
+    pub secret: Option<String>,
+    #[clap(long = "relayer")]
+    pub relayer: Option<String>,
+    #[clap(long = "ip", default_value = "0.0.0.0")]
+    pub ip: IpAddr,
+    #[clap(long = "port", default_value_t = DEFAULT_PORT)]
+    pub port: u16,
+    #[clap(long = "peering-port", default_value_t = DEFAULT_PEERING_PORT)]
+    pub peering_port: u16,
+    #[clap(long = "db-path", default_value = default_testnet_db_path().into_os_string())]
+    pub db_path: PathBuf,
+    #[clap(long = "bootstrap-node", default_value_t = TESTNET_BOOTSTRAP_NODE.to_string())]
+    pub bootstrap_node: String,
```

Comment:
nit: these are the same as the default long arg, so it's not necessary to specify them.

```suggestion
    #[clap(long)]
    pub peer_id: Option<String>,
    #[clap(long)]
    pub secret: Option<String>,
    #[clap(long)]
    pub relayer: Option<String>,
    #[clap(long, default_value = "0.0.0.0")]
    pub ip: IpAddr,
    #[clap(long, default_value_t = DEFAULT_PORT)]
    pub port: u16,
    #[clap(long, default_value_t = DEFAULT_PEERING_PORT)]
    pub peering_port: u16,
    #[clap(long, default_value = default_testnet_db_path().into_os_string())]
    pub db_path: PathBuf,
    #[clap(long, default_value_t = TESTNET_BOOTSTRAP_NODE.to_string())]
    pub bootstrap_node: String,
```